doc/report-types.md

```diff
@@ -0,0 +1,41 @@
+# Scope of this document
+The service is currently conceived in such a way to allow any combination of filters options with any of the types to group by.
+However the UI bundles 6 of these combinations as PublicationReportTypes. This document describes the matching filter options and group option by for each PublciationReportType.
+Because a change might be required that restricts the combinations, this document documents the combinations in use by the frontend. When we choose a less dynamic approach, it might be useful to consider grouping and preset filters can be moved to SPARQL directly.
+
+In order to understand the filter options and group options, please consult the **Reference** section [README.md](../README.md).
+
+In order to add a PublicationReportType define it in [app-kaleidos](https://github.com/kanselarij-vlaanderen/app-kaleidos)
```

Comment:
Perhaps you could mention where to specify it here. Would save a dev a couple of minutes looking for it.
I would add the following:

The specification for the model is here in `config/resources/job-domain.lisp`, but to add a PublicationRapportType, a migration needs to be added, similar to `config/migrations/20220511133621-add-publication-report-types.ttl` in [app-kaleidos](https://github.com/kanselarij-vlaanderen/app-kaleidos)